Makefile

```diff
@@ -162,7 +165,7 @@ e2e:
 
 # end-to-end-test for testing upgrading
 .PHONY: e2e-upgrade
-e2e-upgrade:
+e2e-upgrade: undeploy
```

Comment:
why did we add an updeploy here?

Reply:
Because the upgrade E2E test will install first the OTEL Operator 0.49.0 as first step. If we left the current operator version installed, we can be altering the environment. I mean: the cluster will not be clean for the E2E test.